packages/venia-ui/lib/drivers/adapter.js

```diff
@@ -60,13 +60,15 @@ const VeniaAdapter = props => {
         const cache = apollo.cache ? apollo.cache : preInstantiatedCache;
         const client = new ApolloClient({ cache, link });
 
+        client.apiBase = apiBase;
+
         return client;
     }, [apiBase, apollo]);
 
     return (
         <ApolloProvider client={apolloClient}>
             <ReduxProvider store={store}>
-                <Router apiBase={apiBase}>{children}</Router>
```

Comment:
Just to clarify we are intentionally not removing the unused `Router` implementation right? We also have some old devdocs for it so I want to make sure we're not leaving outdated references lying around.

Reply:
Correct. I didn't delete any of the old implementations since it wasn't strictly necessary to achieve the goal here. So we have a choice:

- Remove unused files, such as `Router` and `MagentoRouteHandler`
- Deprecate unused files, but remove them later on